client/src/index.js

```diff
@@ -1,26 +1,49 @@
 import Phaser from 'phaser';
-import logoImg from './assets/logo.png';
+import { Player } from './objects/player';
+import playerImg from './assets/player_image.png';
+
 
 class Game extends Phaser.Scene {
     constructor() {
         super();
+
+        this.cursors = {}
+        this.player = null
+        this.wasd = {}
     }
 
     preload() {
-        this.load.image('logo', logoImg);
+        this.load.image('player', playerImg);
     }
 
     create() {
-        const logo = this.add.image(400, 150, 'logo');
-
-        this.tweens.add({
-            targets: logo,
-            y: 450,
-            duration: 2000,
-            ease: "Power2",
-            yoyo: true,
-            loop: -1
-        });
+
+        this.player = new Player(this, 300, 300, 'player');
+        this.add.existing(this.player);
```

Comment:
This can probably be removed because we already add `player` to the physics engine in it's own class

Reply:
If removed the object is not rendered, only used for physics.